MyOwnORM/MyOwnORM/CustomDbContextOptionsBuilder.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbContextOptionsBuilder 
+    {
+        public static void UseSqlServer(string connectionString)
```

Comment:
I don't understand how it will work only for sql server. If for example I set connection string for Postgre here, it will work. So, extend this to use only sql server or rename to 'TestConnection' as example

Reply:
Changed to 'TestConnection'